### PR TITLE
Update TelemetryClient dependency to match Main Panopto Solution

### PR DESCRIPTION
**Pull request by vchen-panopto**
In preparation to add TelemetryClient binary to Main Panopto solution, update all AWS binary dependency versions to match the main solution.

### CODE REVIEW

File: AWSTelemetryClient/AWSTelemetryClient.csproj [hunk 1]
```diff
@@ -21,6 +21,7 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <LangVersion>5</LangVersion>
```
Comment (bmartin-pan) [inline AWSTelemetryClient/AWSTelemetryClient.csproj]: Doesn't this library use interpolated strings which requires C# 6?

File: TelemetryClient/TelemetryClient.csproj [hunk 2]
```diff
@@ -21,6 +21,7 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <LangVersion>5</LangVersion>
```
Comment (bmartin-pan) [inline TelemetryClient/TelemetryClient.csproj]: Doesn't this library use interpolated strings which requires C# 6?

File: AWSTelemetryClient/AWSTelemetryClient.csproj (hunk 1 above)
Comment (vchen-panopto) [inline AWSTelemetryClient/AWSTelemetryClient.csproj]: Fixed. Enabled treat warning as error and bumped LangVersion from 5 -> 6

File: TelemetryClient/TelemetryClient.csproj (hunk 2 above)
Comment (vchen-panopto) [inline TelemetryClient/TelemetryClient.csproj]: Fixed. Enabled treat warning as error and bumped LangVersion from 5 -> 6

**Pull request closed without merging**